modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -49,6 +56,8 @@ def is_module_options_ready?(mod)
   end
 
   def is_module_platform?(mod)
+    return true if datastore["IGNOREARCH"] == true
```

Comment:
```suggestion
    return true if datastore["IgnoreArch"] == true
```

Similar change should be done just below for `IgnoreIncompatible` 👀